apps/studio/src/features/editing-experience/components/form-builder/FormBuilder.tsx

```diff
@@ -26,6 +28,7 @@ import {
 } from './renderers'
 
 const renderers: JsonFormsRendererRegistryEntry[] = [
+  { tester: jsonFormsObjectControlTester, renderer: JsonFormsObjectControl },
```

Comment:
Will this be used by all components that define a object?

it's actually very specifc to objects which contain string-type text fileds with limited allowable values (enums) right?

Reply:
Hmm no, it's really all kinds of objects (i.e. with `type` set to `object`). The actual rendering of individual property fields are done via `JsonFormsDispatch`.